tests/enum.rs

```diff
@@ -6,32 +6,34 @@ winrt::import!(
         "windows.application_model.appointments"
 );
 
+use winrt::RuntimeType;
+
 #[test]
 fn signed_enum() {
     use windows::foundation::AsyncStatus;
 
-    assert!(AsyncStatus::default() == AsyncStatus::Canceled);
-    assert!(AsyncStatus::Canceled as i32 == 2);
-    assert!(AsyncStatus::Completed as i32 == 1);
-    assert!(AsyncStatus::Error as i32 == 3);
-    assert!(AsyncStatus::Started as i32 == 0);
+    assert!(AsyncStatus::default().abi() == 0);
```

Comment:
I think it would make sense to implement PartialEq for these for the type that represents the enum